index.html

```diff
@@ -12,9 +12,17 @@
     </head>
 
     <body>
+        <div class="menu">
+            <div class="nav-item"><a href="#about_sec">about</a></div>
```

Comment:
I would capitalize the A in 'about' here to keep it consistent with the rest of your nav